### DISCUSSION TITLE
Question Difficulty ‌Btc

### DISCUSSION THREAD

**Discussion by 1984winstonsmith** [coinfoundry/miningcore#1188] [General]
Hello, considering the difficulty of the bitcoin network, which is 27 th/s, do I receive some bitcoins daily using this software and 20 Hashrate?

**Comment by oliverw** [comment]
20 what?

**Comment by 1984winstonsmith** [comment]
> 20 what?

20 Th power of miner

**Comment by 1984winstonsmith** [comment]
I mean, with a miner whose processing power is 20 th / s, can I make a profit by launching this pool daily or should I wait for a block on the pool to be solved by the miner?

**Comment by oliverw** [comment]
You won't earn anything for a loooong time as you would be solo-mining BTC.

**Comment by 1984winstonsmith** [comment]
Thanks
Can I pay for miners per week with 3 Ph / s?

**Comment by seperatis** [comment]
> Thanks Can I pay for miners per week with 3 Ph / s?

Even if you rented 3 PH/s it would take you 463 days at 100% Effort to solve a Bitcoin block. Yes, you might get Lucky and solve it sooner.... Or you might have bad luck and take 900+ days instead.

As @oliverw has already said, you would be effectively SOLO mining, and it would take 100 years+ with only 20 TH/s to ever see a block solved.

**Comment by 1984winstonsmith** [comment]
Thanks
Do pools with less than 3 pay for minor mines out of their own pockets? Until the block is found?

> > Thanks Can I pay for miners per week with 3 Ph / s?
> 
> Even if you rented 3 PH/s it would take you 463 days at 100% Effort to solve a Bitcoin block. Yes, you might get Lucky and solve it sooner.... Or you might have bad luck and take 900+ days instead.
> 
> As @oliverw has already said, you would be effectively SOLO mining, and it would take 100 years+ with only 20 TH/s to ever see a block solved.

Thanks
Do pools with less than 3 pay for minor mines out of their own pockets? Until the block is found?

**Comment by seperatis** [comment]
@1984winstonsmith

What pools with less than 3 PH? LOL....
There aren't any......not that are solving blocks regularly anyways...

Also, this is not a bug. This is a question and it should've been posted in the Discussion section, opening up an issue is for bugs or problems related to this software itself, not general questions. Thanks.

**Reply by 1984winstonsmith** [reply]
Thanks.

**Comment by oliverw** [comment]
Migrating this to a discussion topic.

**Reply by 1984winstonsmith** [reply]
> Migrating this to a discussion topic.

sorry, Thanks.

**Reply by oliverw** [reply]
Why sorry. Its a legitimate question.
